lib/queue/events.js

```diff
@@ -66,13 +66,12 @@ exports.subscribe = function() {
   if( exports.subscribeStarted ) return;
   var client    = redis.pubsubClient();
   client.on('message', exports.onMessage);
-  client.on('subscribe', function() {
+  client.subscribe(client.getKey(exports.key),  function() {
```

Comment:
Is this backward compatible? node_redis officially hasn't documented the callback on the .subscribe method.